esx_service/cli/vmdkops_admin_test.py

```diff
@@ -379,6 +410,371 @@ class TestStatus(unittest.TestCase):
     def test_status(self):
         self.assertEqual(vmdkops_admin.status(None), None)
 
+class TestTenant(unittest.TestCase):
+    """ Test tenant functionality """
+    # tenant1 info
+    tenant1_name = "test_tenant1"
+    vm1_name = 'test_vm1'
+    vm1 = None
+    vm2_name = 'test_vm2'
+    vm2 = None
+    tenant1_new_name = "new_test_tenant1"
+    datastore_name = None
+    datastore1_name = None
+
+    def setUp(self):
+        """ Setup run before each test """
+        
+        if (not self.datastore_name):
+            datastores = vmdk_utils.get_datastores()
+            if datastores:
+                datastore = datastores[0]
+                self.datastore_name = datastore[0]
+                self.datastore_path = datastore[2]
+            
+                if len(datastores) >= 1:
```

Comment:
sorry for the confusion...

let's say we have only one datastore ```if len(datastores) >= 1:``` satisfies and failed at line:443. 

> Those test will be skipped if the condition is not met.

Can you please point me to the line which skips the TestTenant when only one datastore is found in the inventory?

Thanks!